piximedia/DOCUMENTATION.md

```diff
@@ -0,0 +1,79 @@
+# Piximedia
+## General Compatibility
+|Feature|  |
+|---|---|
+| Consent | Yes |
+| Native Ad Support | No |
+| SafeFrame Support | Yes |
+| PMP Support | No |
+ 
+## Browser Compatibility
+| Browser |  |
+|--- |---|
+| Chrome | Yes |
+| Edge | Yes |
+| Firefox | Yes |
+| Internet Explorer 9 | No |
+| Internet Explorer 10 | No |
+| Internet Explorer 11 | Yes |
+| Safari | Yes |
+| Mobile Chrome | Yes |
+| Mobile Safari | Yes |
+| UC Browser | Yes |
+| Samsung Internet | Yes |
+| Opera | Yes |
+ 
+## Adapter Information
+| Info | |
+|---|---|
+| Partner Id | PiximediaHtb |
+| Ad Server Responds in (Cents, Dollars, etc) | Euros |
+| Bid Type (Gross / Net) | Net |
+| GAM Key (Open Market) | ix_pix_cpm |
+| GAM Key (Private Market) | ix_pix_cpm |
+| Ad Server URLs | //ad.piximedia.com/hbie |
+| Slot Mapping Style (Size / Multiple Sizes / Slot) | Slot |
+| Request Architecture (MRA / SRA) | SRA |
+ 
+## Currencies Supported
+ 
+## Bid Request Information
+### Parameters
+| Key | Required | Type | Description |
```

Comment:
This table is not well formatted. Please update it to be a proper table.

Reply:
Hello @ix-certification, so sorry about this. Thanks for reporting it, I just updated the PR. 
Let me know if there is anything I can do to assist,
Christopher